### ISSUE TITLE
Don't show white border

### ISSUE THREAD

**Issue by gaearon** [bvaughn/react-devtools-experimental#21]
I think it's a bit weird how component name gets trimmed vertically by an opaque white border at the top.

<img width="308" alt="screen shot 2019-02-28 at 5 25 06 pm" src="https://user-images.githubusercontent.com/810438/53585443-d90d7780-3b7d-11e9-9e32-3c3d7ee7dad8.png">

Can we remove it to match how scroll views usually work?

**Comment by gaearon** [comment]
To be clear I'd expect something like this:

![Uploading Screen Shot 2019-02-28 at 5.28.00 PM.png…]()

Note how there's no white padding at the scrollview border.

**Comment by bvaughn** [comment]
This is a little difficult to do with a windowing lib in a way that doesn't add overhead. You either need to add a special row to the beginning and end of each list that pretends to be padding/margin (in which case you need to use the slightly less efficient variable size list) or you need the list abstraction to support the concept of padding/offset within itself internally (which is something I've pushed back on in react-window for reasons I'd be happy to talk about but am too lazy to write up here again).

Anyway, long story short to just say that this is not an easy fix, and it's probably a bit more nuanced feedback that I'm looking for at this stage in the re-write 😅  so I'm probably not going to take any action on this particular issue (at least for the time being). Maybe once the extension is more feature complete we can revisit UI nits like this though. 😄
